setup_ccd.py

```diff
@@ -191,6 +191,22 @@ class ColumnInfo:
         ],
         alternative="model_Cartn_z",
     ),
+    "alt_atom_id": ColumnInfo(
```

Comment:
Could you move this block directly below the one for `atom_id`? This way it would be structured more logically, I think.

Reply:
Good point, will do!